test/unit/spec/VerifyPIV_spec.js

```diff
@@ -127,6 +132,44 @@ function (Q, Okta, OktaAuth, Util, PivForm, PrimaryAuthForm, Beacon, Expect,
           );
         });
       });
+      itp('shows generic error message for empty error response', function () {
+        var res = deepClone(resError);
+        res.response = {};
+        return setup(res).then(function (test) {
+          return Expect.waitForFormError(test.form, test);
+        }).then(function (test) {
+          expect(test.form.hasErrors()).toBe(true);
+          expect(test.form.errorBox()).toHaveLength(1);
+          expect(test.form.errorMessage())
+            .toEqual('We found some errors. Please review the form and make corrections.');
+        });
+      });
+      itp('shows generic error message for empty text error response', function () {
+        var res = deepClone(resError);
+        res.responseType = 'text';
+        res.response = '';
+        return setup(res).then(function (test) {
+          return Expect.waitForFormError(test.form, test);
+        }).then(function (test) {
+          expect(test.form.hasErrors()).toBe(true);
+          expect(test.form.errorBox()).toHaveLength(1);
+          expect(test.form.errorMessage())
+            .toEqual('We found some errors. Please review the form and make corrections.');
```

Comment:
same question here. 
Or if this is correct and I am missing something, then we need another test for that new error message